src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1074,9 +1075,8 @@ sub check_star_catalog {
     my $min_z = $z_ang_max;
 
     my $dither;			# Global dither for observation
-    if (defined $self->{DITHER_Y_AMP} and defined $self->{DITHER_Z_AMP}) {
-	$dither = ($self->{DITHER_Y_AMP} > $self->{DITHER_Z_AMP} ?
-		   $self->{DITHER_Y_AMP} : $self->{DITHER_Z_AMP}) * 3600.0;
+    if (defined $self->{cmd_dither_y_amp} and defined $self->{cmd_dither_z_amp}) {
+	$dither = max($self->{cmd_dither_y_amy}, $self->{cmd_dither_z_amp});
```

Comment:
Yes, that's correct.  The current result should be conservative, so I think that in most cases we can live with this.  If we get in trouble with spoilers then we'll see.  Do you have a simple plan in mind to implement the correct behavior?